amp/repository/aim/view/scripts/reportWizard/fundingGroups.js

```diff
@@ -15,7 +15,7 @@ YAHOO.amp.reportwizard.fundingGroups["donor"]= new Array(
 				,'Donor Group'
 				,'Donor Agency'
 				,'Primary Sector'
```

Comment:
I know it was hardcoded but lets take the chance to move admin levels to a constant

Reply:
I don't think it is ok to put some literals hardcoded and some of them as constant. Or we put all of them in constants (that will be used only in this js file) or we don't put at all.